File: Week4/Day5/Solutions/C/prog2.c
int printf(const char*,...);
int scanf(const char*,...);


void main() {

	int i = 0,n,temp=0;
	char arr[10];
	printf("Enter Decimal number:");
	scanf("%d",&n);
    	while(n!=0) 
    	{    
        	temp = n%16; 
        	if(temp < 10) 
        	{ 
            		arr[i] = temp + 48; 
            		i++; 
        	} 
        	else
        	{ 
            		arr[i] = temp + 55; 
            		i++; 
        	} 
        	n /= 16; 
    	} 
    for(int j=i-1; j>=0; j--) 
        printf("%c",arr[j]);
   printf("\n"); 
}

File: Week4/Day5/Solutions/Python/prog2.py
num = int(input("Decimal number:"))
ans = ""
while num:
    rem = num%16
    num//=16
    if rem < 10:
        ans+=str(rem)
    elif rem == 10:
        ans+="A"
    elif rem == 11:
        ans+="B"
    elif rem == 12:
        ans+="C"
    elif rem == 13:
        ans+="D"
    elif rem == 14:
        ans+="E"
    elif rem == 15:
        ans+="F"
print(ans[::-1])
